source/fuzz/fuzzer_pass_apply_id_synonyms.cpp

```diff
@@ -91,7 +91,12 @@ void FuzzerPassApplyIdSynonyms::Apply() {
           // Exclude the fact that the id is synonymous with itself.
           continue;
         }
-        synonyms_to_try.push_back(data_descriptor);
+
+        if (DescriptorsHaveCompatibleTypes(
```

Comment:
Descriptors -> DataDescriptors

Reply:
(Partly because "descriptor" is such an overloaded term.)

Reply:
Fixed in 2955d7a.